### PR TITLE
Add libsvm recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by 183amir** [comment]
Anybody wants to be a co-maintainer, please let me know.

File: recipes/libsvm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  name: libsvm
+  version: {{ version }}
+
+source:
+  fn: libsvm-{{ flatversion }}.tar.gz
+  url: https://github.com/cjlin1/libsvm/archive/v{{ flatversion }}.tar.gz
+  md5: acff2c058b7cb38a135fa5d0b4f39e43
+
+build:
+  number: 0
```
Comment (ocefpaf) [inline recipes/libsvm/meta.yaml]: I believe you'll need `features` to build this on Windows. See the gdal recipe as an example:

https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/meta.yaml#L17-L26

Comment (183amir) [inline recipes/libsvm/meta.yaml]: Why would I add a dependency to `python`?

Comment (ocefpaf) [inline recipes/libsvm/meta.yaml]: Good question: https://github.com/conda/conda/wiki/VC-features

**Comment by 183amir** [comment]
Can I not compile this on Windows and just copy the compiled 64-bit library that is provided?
I have no means to test this on Windows and if I do try and error commits, it will pollute the our CI workers.

**Comment by ocefpaf** [comment]
> Can I not compile this on Windows and just copy the compiled 64-bit library that is provided?

We prefer to build from source as much as possible as copying binaries compromises the provenance of the builds.

> I have no means to test this on Windows and if I do try and error commits, it will pollute the our CI workers.

That is not a big deal. However, if you do not have imediate use for the Windows builds you can skip it now and iterate later on the feedstock.

**Comment by patricksnape** [comment]
Wait on this - Have a PR pending on bioidap.

**Comment by jakirkham** [comment]
Thanks for doing that @patricksnape.

**Comment by 183amir** [comment]
@patricksnape could you please fix the Linux builds too and create a new PR?

**Comment by patricksnape** [comment]
@183amir That should be it.

**Comment by 183amir** [comment]
I don't think you should have removed `requirements:
-  build:
-    - python
+test:`
https://github.com/conda-forge/staged-recipes/pull/352#discussion_r59611188

**Comment by patricksnape** [comment]
I have pinged @msarahan already about this - but that is definitely not correct given how conda-build currently works. Python is not required at build time for this recipe, only the correct visual studio feature needs to be activated. This already happens correctly without requiring Python as a build time requirement (as conda-build uses ``CONDA_PY``/``--py``). In fact, what it really needs is Python (or something that activates the correct `vc*` feature) at test time - which this recipe already has. I can confirm that with the latest conda-build this builds on my machine. Also, the jpeg PR is currently failing for this very reason, and it has the build time requirement, which is clearly not sufficient.

**Comment by 183amir** [comment]
In that wiki says:
```
Presently, only the Python package tracks features. It does so with code like (example from Python 3.5):

 build:
     track_features:
         - vc14           [win]

For this reason, any code that depends on Python also tracks this feature. Different versions of Python track different features, as appropriate. As a result, you'll see recipes sometimes depending on Python as a build-time dependency when Python isn't actually used to build a package:
```
Now, I am not very expert on this. I think the best way would be if you open a new pull request here on conda-forge with what we already have and discuss it further on the new pull and test it against appveyor. This way, it would be easier to test windows builds without waiting for me.

**Comment by jakirkham** [comment]
Continued in this PR ( https://github.com/conda-forge/staged-recipes/pull/358 ).

**Pull request closed without merging**